ui/app/clinical/common/views/visitSummary.html

```diff
@@ -28,65 +25,79 @@
                     </section>
                 </div>
             </div>
-            <div ng-switch-when="diagnosis">
-                <section ng-if="::section.config" class="block dashboard-section">
-                    <bahmni-diagnosis patient-uuid="::section.config.patientUuid"
-                                      config="::section.config" visit-uuid="::visitUuid" show-latest-diagnosis=
-                                              "true"></bahmni-diagnosis>
+            <div ng-switch-when="vitals" class="section">
+                <div ng-if="::section.config" id="vital-section">
+                    <section class="block observation dashboard-section" id="vitalSection">
+                        <ul bm-gallery access-impression="true">
+                            <bahmni-observation patient="::patient" visit-uuid="{{::visitUuid}}"
+                                                config="::section.config"
+                                                section-title="::section | titleTranslate"
+                                                class="section-title"></bahmni-observation>
+                        </ul>
+                    </section>
+                </div>
+            </div>
+            <div ng-switch-when="consultationNotes" class="section">
+                <div ng-if="::section.config" id="consultationNotes-section">
+                    <section class="block observation dashboard-section" id="consultationNotesSection">
+                        <ul bm-gallery access-impression="true">
+                            <bahmni-observation patient="::patient" visit-uuid="{{::visitUuid}}"
+                                                config="::section.config"
+                                                section-title="::section | titleTranslate"
+                                                class="section-title"></bahmni-observation>
+                        </ul>
+                    </section>
+                </div>
+            </div>
+            <div ng-switch-when="diagnosis" class="section">
+                <section ng-if="::section.config" class="block dashboard-section section-title has-link">
+                    <bahmni-diagnosis patient-uuid="::patient.uuid" config="::section" show-latest-diagnosis="false"
+                                      show-ruled-out-diagnoses="::section.showRuledOutDiagnoses"></bahmni-diagnosis>
```

Comment:
is there a requirement for showing ruled out diagnosis?

Reply:
Not really. Just need to print Confirmed/Suspected diagnosis in print out.